src/nagendra/BasicElementsPageAlert.java

```diff
@@ -0,0 +1,121 @@
+		driver.findElement(By.xpath("//a[text()='Basic Elements']")).click();
+		WebElement alertButton = driver.findElement(By.xpath("//button[text()='Alert']"));
+		JavascriptExecutor js = (JavascriptExecutor) driver;
+		js.executeScript("arguments[0].scrollIntoView(true)", alertButton);
+		System.out.println("Click on Alert Button");
+		alertButton.click();
+		System.out.println("STEP- Switch to Alert on Alert Button");
+		Alert alert = driver.switchTo().alert();
+		String message = alert.getText();
+		System.out.println("Message is" + message);
+		System.out.println("STEP-Accept the alert");
+		alert.accept();
+		String expectedMessage = "You must be TechnoCredits student!!";
+		if (message.equals(expectedMessage)) {
+			System.out.println("The Alert Message " + message + " Matches");
+		} else {
+			System.out.println("Both The Messages are different ");
+		}
+	}
+
+	public static void VerifyJavaScriptConfirmationButtonLabelMessage(WebDriver driver) {
+		System.out.println("STEP - Applied implicit wait to driver instance");
+		driver.manage().timeouts().implicitlyWait(5000, TimeUnit.MILLISECONDS);
+		System.out.println("STEP -Click on Basic Elements  Page ");
+		driver.findElement(By.xpath("//a[text()='Basic Elements']")).click();
+		WebElement alertButton = driver.findElement(By.xpath("//button[text()='Javascript Confirmation']"));
+		JavascriptExecutor js = (JavascriptExecutor) driver;
+		js.executeScript("arguments[0].scrollIntoView(true)", alertButton);
+		System.out.println("Click on Alert Button");
+		alertButton.click();
+		System.out.println("STEP- Switch to Alert on Alert Button");
+		Alert alert = driver.switchTo().alert();
+		String message = alert.getText();
+		System.out.println("Message is" + message);
+		System.out.println("STEP-Accept the alert");
+		alert.accept();
+		WebElement label = driver.findElement(By.xpath("//p[@id='pgraphdemo']"));
+		String OkLabelMessage = label.getText();
+		String expectedMessage = "You pressed OK!";
+		if (OkLabelMessage.equals(expectedMessage)) {
+			System.out.println("PASSED- Label Message is Same for Ok");
+		} else {
+			System.out.println("FAILED- Label Message is different");
+		}
+		String cancelLabelMessage = "You pressed Cancel!";
+		System.out.println("Click on Alert Button");
+		alertButton.click();
+		System.out.println("STEP- Switch to Alert on Alert Button");
+		driver.switchTo().alert();
+		System.out.println("STEP-Dismiss the alert");
+		alert.dismiss();
+		String cancelMessage = label.getText();
+		if (cancelMessage.equals(cancelLabelMessage)) {
+			System.out.println("PASSED- Label Message is Same For Cancel");
+		} else {
+			System.out.println("FAILED- Label Message is different");
+		}
+	}
+
+	public static void VerifyJavaScriptPromptButtonLabelMessage(WebDriver driver) {
+		System.out.println("STEP - Applied implicit wait to driver instance");
+		driver.manage().timeouts().implicitlyWait(5000, TimeUnit.MILLISECONDS);
```

Comment:
Use only one time implicitly wait as you are passing same driver to every method.